docs/v3.x/plugins/email.md

```diff
@@ -124,7 +127,10 @@ If you're using a different provider depending on your environment, you can spec
 :::
 
 ::: tip
-When testing the new email provider with one the two templates created during strapi setup, the shipper email used is the one on the template, which is no-reply@strapi.io and it has to be modified to the email used with your email provider, otherwise it will fail the test.
+Only one email provider will be active at all time. If the email provider setting isn't picked up by strapi, verify you have put the file `plugins.js` in the correct folder, and with correct filename. The selection of email provider is done via configuration file only.               
+
+::: tip
```

Comment:
Should this be two tips or one joined one? if it's the former we need to close out the top tip with the `:::` before adding another.

Reply:
I would like to propose to be two tips. 
Yup, agreed that with the changes above to include the `:::` as closing syntax.

Reply:
added new commit addressing to this.